apps/www/styles/index.css

```diff
@@ -8,6 +8,33 @@
 @import './../../../packages/ui/build/css/themes/dark.css';
 @import './../../../packages/ui/build/css/themes/light.css';
 
+@layer components {
+  .focus-base {
```

Comment:
Could put these into the tailwind config maybe.
Don't have to do it in this PR

Reply:
That would be ideal, but I couldn't figure out a way to use raw tailwind classes to create a new class 🤔

Like:

```js
extend: {
  focus: {
    base: '@apply focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-scale-900 ring-0 transition-all focus-visible:ring-offset-transparent',
  }
}
```

Any idea?

Reply:
Need to use a utility for a composition of tailwind.

Could use matchUtilities
https://tailwindcss.com/docs/plugins#dynamic-utilities

Or maybe make a utility just to reset all the focus stuff, so all you need then is something like `focus-reset focus-visible:ring-scale-900`